### PR TITLE
Error success message for org email update

### LINKED ISSUE #1311: User friendly message should be displayed if the email update for an Org is successful or NOT.
1. Open "https://staging.fightpandemics.work/"
2. Login using valid email/pwd
3. Navigate to Profile -> Org_name[Already added]
4. Click on EDIT icon to edit account information
5. Update the 'Organisation email" to existing email in the system, say for e.g. try with the individual user email ID and click on 'Save Changes' button.

Issue#: You will observe that nothing will happen and only the focus moves to the top of the screen and revert the changes.
Refer below GIF.
https://share.getcloudapp.com/2NuyK5X1

6. Now update the email with valid unique email ID and click on 'Save Changes' 
You will observe that email for the Org is updated and user is directed to Org Profile page.

Expected behavior: Ideally, user should get user friendly message if the email update is successfully or unsuccessful.

### PR DESCRIPTION
**Pull request by soniaroh**
##### created success and error user friendly message for when user updates org email.
Screen Shot 2020-08-11 at 10 48 43 PM

ezgif com-video-to-gif (1)

closes #1311 
_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix. The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/pages/EditOrganisationAccount.js [hunk 1]
```diff
@@ -320,6 +367,17 @@ function EditOrganisationAccount(props) {
...
             Edit Organisation Profile
           </CustomEditAccountHeader>
+          {isEmailUpdateSuccess ? (
+            <>
+              <SuccessAlert message={"Email successfully updated"} />
+            </>
+          ) : isEmailUpdateError ? (
+            <>
+              <ErrorAlert message={orgProfileState.error} />
+            </>
```
Comment (adisen) [inline client/src/pages/EditOrganisationAccount.js]: I think you should probably add the Fragment tag. I think it's better.

**Review by adisen — APPROVED** [review]
LGTM.

File: client/src/pages/EditOrganisationAccount.js (hunk 1 above)
Comment (idjevm) [reply client/src/pages/EditOrganisationAccount.js]: No need though since we are mostly using the short version (which is these empty tags)

**Review by idjevm — CHANGES_REQUESTED** [review]
I am not sure about the design of this. Could you double-check with the design team? Please include some images of what the changes are so they can review here, since Build URLs become inactive after 24hrs, also please link the proper issue.

**Comment by soniaroh** [comment]
@rabbi08 @ppagarawal  can you guys please check the design and let me know if any changes needs to be made?

**Comment by yu-zhao-xyz** [comment]
Using the design from the current design system: for confirmation please check https://zpl.io/bJJMeDK, for error please check https://zpl.io/bJJMeDK
Preview here: 

image

**Comment by ppagarawal** [comment]
> Using the design from the current design system: for confirmation please check https://zpl.io/bJJMeDK, for error please check https://zpl.io/bJJMeDK
> Preview here:
> 
> image

@nr431207 Please use the red text in case of error. For success messages, we can have a small overlay window for 10 seconds on active tab (and then fade away effect) and a close button on successful update. 

I hope that clarifies the requirement

**Comment by soniaroh** [comment]
> > Using the design from the current design system: for confirmation please check https://zpl.io/bJJMeDK, for error please check https://zpl.io/bJJMeDK
> > Preview here:
> > image
> 
> @nr431207 Please use the red text in case of error. For success messages, we can have a small overlay window for 10 seconds on active tab (and then fade away effect) and a close button on successful update.
> 
> I hope that clarifies the requirement

Ive added some changes, please review, thanks! @ppagarawal

**Comment by ppagarawal** [comment]
> > > Using the design from the current design system: for confirmation please check https://zpl.io/bJJMeDK, for error please check https://zpl.io/bJJMeDK
> > > Preview here:
> > > image
> > 
> > @nr431207 Please use the red text in case of error. For success messages, we can have a small overlay window for 10 seconds on active tab (and then fade away effect) and a close button on successful update.
> > 
> > I hope that clarifies the requirement
> 
> Ive added some changes, please review, thanks! @ppagarawal

Amazing work 👌 as always @nr431207 !

**Review by idjevm — APPROVED** [review]
I think this looks great. But I think if we want to keep consistency, we would want this success message to display anytime that the `Save Changes` button is clicked for any field, and not only for the email though. 

Any thoughts @ppagarawal? - if so please create or add additional info to this ticket so they can be addressed here. I wouldn't want to have this message alone only for one field on prod.

**Review by idjevm — CHANGES_REQUESTED** [review]
@nr431207 So ideally, to keep consistency, we would want to have this message whenever the user clicks on `Save changes`. I think there are some discussions needed. Since sometimes we just redirect the user to the profile, like in the user profile and org profile too. the logic is kinda the same so it would make sense to keep consistency across both org profile and user profile...

@ppagarawal up to you what if you would want to create smaller new tickets to make sure the concistency is kept.

**Comment by nr431207** [comment]
> @nr431207 So ideally, to keep consistency, we would want to have this message whenever the user clicks on `Save changes`. I think there are some discussions needed. Since sometimes we just redirect the user to the profile, like in the user profile and org profile too. the logic is kinda the same so it would make sense to keep consistency across both org profile and user profile...
> 
> @ppagarawal up to you what if you would want to create smaller new tickets to make sure the concistency is kept.

@idjevm @ppagarawal the related issue was focused on email update, but it makes sense to have this message to show up for any filed update, I feel it's better to create a separate ticket for that, thoughts?

**Comment by nr431207** [comment]
> @nr431207 So ideally, to keep consistency, we would want to have this message whenever the user clicks on `Save changes`. I think there are some discussions needed. Since sometimes we just redirect the user to the profile, like in the user profile and org profile too. the logic is kinda the same so it would make sense to keep consistency across both org profile and user profile...
> 
> @ppagarawal up to you what if you would want to create smaller new tickets to make sure the concistency is kept.

**Comment by soniaroh** [comment]
@ppagarawal applied success message for all field updates , if user clicks on save without changing anything, user is redirected to org profile.
FightPandemics (4) (1)

**Review by idjevm — APPROVED** [review]
Sweet! Thanks Sonia! I will merge this and reiterate if needed!

**Pull request merged**